src/main/kotlin/org/jitsi/nlj/util/ClockUtils.kt

```diff
@@ -25,6 +25,9 @@ import org.jitsi.utils.TimeUtils
 @JvmField
 val NEVER: Instant = Instant.MIN
 
+@JvmField
+val UNKNOWN: Duration = Duration.ofSeconds(Long.MIN_VALUE)
```

Comment:
For some reason this doesn't feel quite right to me as definition for "unknown" (even in comparison to using `Instant.MIN` for `NEVER`).  What about just using a nullable type for the latest RTT in TransportCcEngine?